$NetBSD: patch-ag,v 1.6 2015/12/29 18:17:04 dholland Exp $

Configure for pkgsrc. Honor COMPILER_RPATH_FLAG.
Use pkgsrc boehm-gc.

XXX: why does this change the version number?

--- System/Common/CONFIG.proto.orig	Wed Nov  3 18:48:17 1999
+++ System/Common/CONFIG.proto
@@ -23,12 +23,12 @@
 ZONES:			false;
 TRACE:			false;
 LIBRARY:		"SATHER_LIBRARY","Library/Library.module";
-VERSION:		"1.2b";
-C_COMPILER:		"gcc ";
-CC_OPTIONS:		"";
-GC_LINK:		"-lgc";
+VERSION:		"1.2.1";
+C_COMPILER:		"@CC@";
+CC_OPTIONS:		"-I@GC_PREFIX@/include/gc";
+GC_LINK:		"-L@GC_PREFIX@/lib @COMPILER_RPATH_FLAG@@GC_PREFIX@/lib -lgc";
 LINK_OPTIONS:		"-lm";
-MAKE_COMMAND:		"make";
+MAKE_COMMAND:		"@GMAKE@";
 OBJECT_EXT:		".o";
 LIB_EXT:		".a";
 C_EXT:			".c";
@@ -42,7 +42,7 @@
 CC_PROLIX_FLAG:		"",""; -- passed to the C compiler (prolix implies verbose)
 MAKE_VERBOSE_FLAG:	"-s","-s"; -- passed to make
 
-FORTRAN_APPEND_UNDERSCORE:	"false";
+FORTRAN_APPEND_UNDERSCORE:	"true";
 FORTRAN_PREFIX_UNDERSCORE:	"false";
 FORTRAN_BIND_FUNC:		"true";
 
